src/data/real/ennreal.lean

```diff
@@ -1337,6 +1337,34 @@ end
 @[simp] lemma one_sub_inv_two : (1:ℝ≥0∞) - 2⁻¹ = 2⁻¹ :=
 by simpa only [div_eq_mul_inv, one_mul] using sub_half one_ne_top
 
+/-- The birational order isomorphism between `ℝ≥0∞` and the unit interval `set.Iic (1 : ℝ≥0∞)`. -/
+def order_iso_Iic_one_birational : ℝ≥0∞ ≃o Iic (1 : ℝ≥0∞) :=
```

Comment:
Can you add simp lemmas for this, either manually or via `simps`?

Reply:
I added some `simp` lemmas.